plugins/legacy-plugin-chart-time-table/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@superset-ui/legacy-plugin-chart-time-table",
+  "version": "0.13.26",
+  "description": "Superset Chart Plugin - Time Table",
+  "sideEffects": [
+    "*.css"
+  ],
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "files": [
+    "esm",
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache-superset/superset-ui.git"
+  },
+  "keywords": [
+    "superset"
+  ],
+  "author": "Superset",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/apache-superset/superset-ui/issues"
+  },
+  "homepage": "https://github.com/apache-superset/superset-ui#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@data-ui/sparkline": "^0.0.84",
+    "@superset-ui/chart-controls": "^0.14.3",
```

Comment:
Let's follow the convention and make `@superset-ui` packages `peerDependencies`.